libraries/AP_Scripting/lua_scripts.h

```diff
@@ -94,6 +94,13 @@ class lua_scripts
     int docall(lua_State *L, int narg, int nres);
     int sandbox_ref;
 
+    enum RELOAD_STATE {
+      RELOAD_NONE = 0,
+      RELOAD_TRUE = 1,
+      RELOAD_DELAY = 2,
+    };
```

Comment:
```suggestion
    enum class RELOAD_STATE {
      NONE = 0,
      TRUE = 1,
      DELAY = 2,
    };
```

Reply:
Also not sure you actually need to specify the values here, let the enum work for you, as these aren't exposed via parameters. There's also been some debate about not caps locking the entrys for a enum class.